java/src/main/java/org/rocksdb/SstFileWriter.java

```diff
@@ -126,10 +128,24 @@ public void put(final DirectSlice key, final DirectSlice value)
    * @throws RocksDBException thrown if error happens in underlying
    *    native library.
    */
-public void put(final byte[] key, final byte[] value)
-    throws RocksDBException {
-  put(nativeHandle_, key, value);
-}
+  public void put(final byte[] key, final byte[] value)
+      throws RocksDBException {
+    put(nativeHandle_, key, value);
+  }
+
+  /**
+   * Add a Put key with value to currently opened file.
+   *
+   * @param key the specified key to be inserted (reading from position to limit).
+   * @param value the value associated with the specified key (reading from position to limit).
+   *
+   * @throws RocksDBException thrown if error happens in underlying
+   *    native library.
+   */
+  public void put(final ByteBuffer key, final ByteBuffer value)
```

Comment:
seems it has been implemented by https://github.com/facebook/rocksdb/commit/e412a426d6ce019fb12f094f64e7f105d227201d . https://github.com/facebook/rocksdb/blob/2e09a54c4fb82e88bcaa3e7cfa8ccbbbbf3635d5/java/src/main/java/org/rocksdb/SstFileWriter.java#L134